docs/maui/extensions/color-conversion-extensions.md

```diff
@@ -223,6 +223,13 @@ using CommunityToolkit.Maui.Extensions.Core;
 Colors.Red.ToCmykaString();
 ```
 
+Depends on the culture settings, **alpha** value may have different delimeter:
+
+```csharp
+new Color(0, 0, 0, 0.5f).ToCmykaString(new System.Globalization.CultureInfo("en-US")); // returns "CMYKA(0%,0%,0%,100%,0.5)"
+new Color(0, 0, 0, 0.5f).ToCmykaString(new System.Globalization.CultureInfo("uk-UA")); // returns "CMYKA(0%,0%,0%,100%,0,5)"
```

Comment:
The converter has this parameter, so it may be expected for users to pass it and get the correct delimiter. For now, XAML doesn't receive this parameter, but it will be added in the future: https://github.com/dotnet/maui/issues/5696

I also propose to add a delimiter property to the converter

Reply:
And also it is a one-way converter

Reply:
discussion on .NET MAUI CommunityToolkit standup: https://youtu.be/Jehqf7B8zzQ?t=2467